### PR TITLE
TELCODOCS-837 KMMO Document new features

### PR DESCRIPTION
**Pull request by slovern**
[TELCODOCS-837](https://issues.redhat.com//browse/TELCODOCS-837) KMMO DOcument new features

Version(s): 4.12

Issue: https://issues.redhat.com/browse/TELCODOCS-837 (RNs for KMM)
https://issues.redhat.com/browse/TELCODOCS-1107 (Removal of TP labels from DTK)

Link to docs preview: 
**Release Notes** https://54462--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-12-release-notes.html#ocp-4-12-scalability-and-performance

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

File: release_notes/ocp-4-12-release-notes.adoc [hunk 1]
```diff
@@ -881,6 +881,28 @@ In {product-title} {product-version}, you can use the {factory-prestaging-tool}
...
 
+[id="ocp-4-12-nfd-multi-namespaces"]
+==== Node Feature Discovery supports multiple namespaces
+
+The Node Feature Discovery (NFD) Operator now supports multiple namespaces.
+
+[id="ocp-4-12-kmmo"]
+==== Kernel Module Management Operator
+
+The Kernel Module Management Operator (KMMO) replaces the Special Resource Operator (SRO).
```
Comment (bthurber) [inline release_notes/ocp-4-12-release-notes.adoc:892]: We are not allowed to capitalize kernel module management.  It must be lower case.

File: release_notes/ocp-4-12-release-notes.adoc [hunk 2]
```diff
@@ -881,6 +881,28 @@ In {product-title} {product-version}, you can use the {factory-prestaging-tool}
...
+[id="ocp-4-12-nfd-multi-namespaces"]
+==== Node Feature Discovery supports multiple namespaces
+
+The Node Feature Discovery (NFD) Operator now supports multiple namespaces.
+
+[id="ocp-4-12-kmmo"]
+==== Kernel Module Management Operator
+
+The Kernel Module Management Operator (KMMO) replaces the Special Resource Operator (SRO). 
+KMMO includes the following features:
```
Comment (bthurber) [inline release_notes/ocp-4-12-release-notes.adoc:893]: Let's change all references of KMMO to KMM.

File: release_notes/ocp-4-12-release-notes.adoc [hunk 3]
```diff
@@ -881,6 +881,23 @@ In {product-title} {product-version}, you can use the {factory-prestaging-tool}
 
 You can now configure OS-level tuning for nodes in a hosted cluster by using the Node Tuning Operator. To configure node tuning, you can create config maps in the management cluster that contain `Tuned` objects, and reference those config maps in your node pools. The tuning configuration that is definied in the `Tuned` objects is applied to the nodes in the node pool. For more information, see xref:../scalability_and_performance/using-node-tuning-operator.adoc#node-tuning-hosted-cluster_node-tuning-operator[Configuring node tuning in a hosted cluster].
 
+[id="ocp-4-12-kmmo"]
+==== Kernel module management operator
```
Comment (Amrita42) [inline release_notes/ocp-4-12-release-notes.adoc:885]:
```suggestion
==== Kernel module management Operator
```

Comment (Amrita42) [reply release_notes/ocp-4-12-release-notes.adoc:885]: https://redhat-documentation.github.io/supplementary-style-guide/#_o

File: release_notes/ocp-4-12-release-notes.adoc [hunk 4]
```diff
@@ -881,6 +881,23 @@ In {product-title} {product-version}, you can use the {factory-prestaging-tool}
...
+
+The kernel module management (KMM) operator replaces the Special Resource Operator (SRO).
```
Comment (Amrita42) [inline release_notes/ocp-4-12-release-notes.adoc:887]:
```suggestion
The kernel module management (KMM) Operator replaces the Special Resource Operator (SRO). 
```

File: release_notes/ocp-4-12-release-notes.adoc [hunk 5]
```diff
@@ -881,6 +881,23 @@ In {product-title} {product-version}, you can use the {factory-prestaging-tool}
...
+[id="ocp-4-12-kmmo"]
+==== Kernel module management operator
+
+The kernel module management (KMM) operator replaces the Special Resource Operator (SRO). 
+KMM includes the following features for connected environments only:
+
+* Hub and spoke support for edge deployments
+* Pre-flight checks for upgrade support
+* Secure boot kernel module signing
+* Must gather to assist with troubleshooting
```
Comment (Amrita42) [inline release_notes/ocp-4-12-release-notes.adoc:893]:
```suggestion
* Must gather logs to assist with troubleshooting
```

File: release_notes/ocp-4-12-release-notes.adoc [hunk 6]
```diff
@@ -881,6 +881,23 @@ In {product-title} {product-version}, you can use the {factory-prestaging-tool}
...
+* Binary firmware deployment
+
+[id="ocp-4-12-hub-and-spoke"]
+==== Hub and spoke cluster support (Technology Preview)
+
+For hub and spoke deployments in an environment that can access the internet, you can use the kernel module management (KMM) operator deployed in the hub cluster to manage the deployment of the required kernel module(s) to one or more managed clusters.
```
Comment (Amrita42) [inline release_notes/ocp-4-12-release-notes.adoc:899]:
```suggestion
For hub and spoke deployments in an environment that can access the internet, you can use the kernel module management (KMM) Operator deployed in the hub cluster to manage the deployment of the required kernel modules to one or more managed clusters. 
```

Comment (Amrita42) [reply release_notes/ocp-4-12-release-notes.adoc:899]: ref https://www.ibm.com/docs/en/ibm-style?topic=punctuation-parentheses

File: release_notes/ocp-4-12-release-notes.adoc (hunk 3 above)
Comment (slovern) [reply release_notes/ocp-4-12-release-notes.adoc:885]: Thanks. Done.

File: release_notes/ocp-4-12-release-notes.adoc (hunk 4 above)
Comment (slovern) [reply release_notes/ocp-4-12-release-notes.adoc:887]: ok

File: release_notes/ocp-4-12-release-notes.adoc (hunk 5 above)
Comment (slovern) [reply release_notes/ocp-4-12-release-notes.adoc:893]: ok

File: release_notes/ocp-4-12-release-notes.adoc (hunk 6 above)
Comment (slovern) [reply release_notes/ocp-4-12-release-notes.adoc:899]: thanks. done

File: hardware_enablement/psap-driver-toolkit.adoc [hunk 7]
```diff
@@ -9,7 +9,6 @@ toc::[]
 Learn about the Driver Toolkit and how you can use it as a base image for driver containers for enabling special software and hardware devices on Kubernetes.
 
 :FeatureName: The Driver Toolkit
-include::snippets/technology-preview.adoc[leveloffset=+0]
```
Comment (mburke5678) [inline hardware_enablement/psap-driver-toolkit.adoc:12]: @slovern This change should to be in a separate PR based off of main.

Comment (slovern) [reply hardware_enablement/psap-driver-toolkit.adoc]: @mburke5678 I think I've fixed that now. The easiest way seemed to be to just add the line back in so there's no change. Does that work for you?
